tests/integration/dekker/Makefile

```diff
@@ -0,0 +1,53 @@
+SPIN?=spin
```

Comment:
This shall be added to `tests/integration/Makefile`

Reply:
Added